src/pointer-analysis/value_set_dereference.cpp

```diff
@@ -77,16 +84,14 @@ exprt value_set_dereferencet::dereference(const exprt &pointer)
     valuet value =
       build_reference_to(*it, pointer, exclude_null_derefs, language_mode, ns);
```

Comment:
This call needs to be amended as it is one of the reasons of the CI failures (Doxygen and clang-format are also unhappy).